### PR TITLE
One jvm per game

### PR DESCRIPTION
**Pull request by kuhlkrein**

### CODE REVIEW

**Comment by kuhlkrein** [comment]
Still in progress: The size of the elements are slower than what they should be in full screen, I need to fix that

**Comment by kuhlkrein** [comment]
SpotBugs error due to plateform.exit(); system.exit(0);

 " **Dm: Method invokes System.exit(…) (DM_EXIT)**
Invoking System.exit shuts down the entire Java virtual machine. This should only been done when it is appropriate. Such calls make it hard or impossible for your code to be invoked by other code. "

any idea on how I could properly close the window and stop the jvm ?

**Comment by kuhlkrein** [comment]
Hello @Tohaker do you know how I could get rid of this DM_EXIT spotbugs error ? Do I have a to add a spacial tag somewhere ? :)

**Comment by Tohaker** [comment]
You'll need to add something to the filter file: https://spotbugs.readthedocs.io/en/stable/filter.html
What's the aim of exiting the JVM here?

**Comment by kuhlkrein** [comment]
@Tohaker it's for fixing issue #894 . Each Time I'm launching a game I'm closing the first JVM and opening a new one

File: gazeplay-core/src/main/java/net/gazeplay/ui/scenes/gamemenu/GameMenuController.java [hunk 1]
```diff
@@ -43,18 +47,62 @@ public void onGameSelection(
...
+        String javaHome = System.getProperty("java.home");
+        String javaBin = javaHome +
+            File.separator + "bin" +
+            File.separator + "java";
+        String classpath = System.getProperty("java.class.path");
+        ProcessBuilder builder;
+        if (gameVariant != null) {
+            builder = new ProcessBuilder(
+                javaBin, "-cp", classpath, GazePlayLauncher.class.getName(),
+                "--user", "Seb",
```
Comment (Tohaker) [inline gazeplay-core/src/main/java/net/gazeplay/ui/scenes/gamemenu/GameMenuController.java:86]: Does this work with `user.name` or something?

File: gradle.properties [hunk 2]
```diff
@@ -1,2 +1,2 @@
-version=1.7.3-SNAPSHOT
+version=1.7.3
```
Comment (Tohaker) [inline gradle.properties:1]: I expect this is for testing purposes only, yes?

**Review by Tohaker — COMMENTED** [review]
I'm aware it's in draft, just wanted to check a few things. Looks good so far though!

File: gazeplay-core/src/main/java/net/gazeplay/ui/scenes/gamemenu/GameMenuController.java (hunk 1 above)
Comment (kuhlkrein) [reply gazeplay-core/src/main/java/net/gazeplay/ui/scenes/gamemenu/GameMenuController.java:86]: Oh yes thank you for this I totally forgot about it

File: gradle.properties (hunk 2 above)
Comment (kuhlkrein) [reply gradle.properties:1]: Idk why but sometimes it changes automatically with gradle build

File: gazeplay-core/spotbugs-exclude.xml [hunk 3]
```diff
@@ -3,6 +3,10 @@
         <Source name="~.*Test\.java" />
         <Bug code="RV, UMAC"/>
     </Match>
+    <Match>
+        <Class name="~.*GameMenuController" />
+        <Bug pattern="DM_EXIT"/>
```
Comment (kuhlkrein) [inline gazeplay-core/spotbugs-exclude.xml:8]: Hello again @Tohaker , I'm doing this, it is supposed to work no ? I don't get why I still have the spotBugs error

Comment (Tohaker) [reply gazeplay-core/spotbugs-exclude.xml:8]: You probably need the file name in the pattern too.

**Comment by Tohaker** [comment]
FYI @kuhlkrein Spotbugs is passing now, your failures were to do with a network issue that github have fixed.

**Comment by kuhlkrein** [comment]
Hello @Tohaker , could you please review the PR ? :)

File: gazeplay-core/src/main/java/net/gazeplay/ui/scenes/gamemenu/GameVariantDialog.java [hunk 4]
```diff
@@ -67,6 +67,8 @@ public GameVariantDialog(
 
         ScrollPane choicePanelScroller = new ScrollPane();
         choicePanelScroller.setContent(choicePane);
+        //choicePanelScroller.setMinHeight(primaryStage.getHeight() / 5);
```
Comment (Tohaker) [inline gazeplay-core/src/main/java/net/gazeplay/ui/scenes/gamemenu/GameVariantDialog.java]: Can we remove commented code please?

File: gazeplay-core/src/main/resources/data/multilinguism/multilinguism.csv [hunk 5]
```diff
@@ -263,6 +263,11 @@ addNewImagethumbup,,,,,,Add new image happy ...,,Ajouter de nouvelles image cont
 addNewImagethumbdown,,,,,,Add new image unhappy...,,Ajouter une nouvelle image pas content ...,,,,,,,,,,Añade nuevas imágenes que no son felices ...,
 addNewImagenocare,,,,,,Add new image don't care...,,Ajouter une nouvelle image s'en fiche ...,,,,,,,,,,Añadir una nueva imagen no importa ...,
 customFileChooser,,,,,,Custom file chooser,,Sélecteur de fichiers personnalisés,,,,,,,,,,,
+Done,,,,,,Done,,Valider,,,,,,,,,,,
+WhereIsItDirectory:,,,,,,'Where is it ?' Configuration:,,Configuration de 'Où est-ce ?':,,,,,,,,,,,
+WhereIsItNotConfigDirectory,,,,,,'Where is it directory' not configured,,'Où est-ce' répertoire non configuré',,,,,,,,,,,
```
Comment (Tohaker) [inline gazeplay-core/src/main/resources/data/multilinguism/multilinguism.csv:268]: English would be better as `'Where is it' directory...`

Comment (kuhlkrein) [reply gazeplay-core/src/main/resources/data/multilinguism/multilinguism.csv:268]: Oh yes there is an error in french too thank you :)

**Review by Tohaker — APPROVED** [review]

**Pull request merged**